src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -32,11 +32,17 @@ public partial class Activity
         /// <summary>
         /// This is an ID that is specific to a particular request.   Filtering
         /// to a particular ID insures that you get only one request that matches.  
-        /// Id has a hierarchical structure: /root-id.id1.id2.id3.  Id is generated when 
+        /// Id has a hierarchical structure: '|root-id.id1_id2.id3_' Id is generated when 
         /// <see cref="Start"/> is called by appending suffix (preceeded with '.') to Parent.Id
         /// or ParentId; Activity has no Id until it started
-        /// See <see href="https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/ActivityUserGuide.md#activity-id"/> for nore details
+        /// <para/>
+        /// See <see href="https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/ActivityUserGuide.md#id-format"/> for more details
         /// </summary>
+        /// <example>Id looks like |RD0003FF21406D-5d183ab6-a000b421.1.8e2d4c28_1.:<para />
+        ///  - 'RD0003FF21406D-5d183ab6-a000b421' - Id of the first, top-most, Activity created; It is a RootId for this and all child Activities<para />
+        ///  - 'RD0003FF21406D-5d183ab6-a000b421.1.' - Id of a child activity. It was started in the same process and ends with '.'<para />
```

Comment:
My understanding is that RD0003FF21406D is the name of the host.   If I am correct here, this is a pretty bad example to use in the explanation because it looks like some sort of generated ID.    Please change it to be something like 'ServerMach35' or something like that to improve readability.